rwkv.cpp

```diff
@@ -1568,6 +1601,16 @@ bool rwkv_eval(struct rwkv_context * ctx, const uint32_t token, const float * st
 
     rwkv_set_inputs(ctx, state_in);
     ggml_set_i32(ctx->serial_graph.tokens, token);
+
+    // Short circuit computation of logits if nobody actually cares
```

Comment:
I suggest
`// Short circuit computation of logits if they are not needed`

Reply:
But my comment is funnier~